### ISSUE TITLE
Cannot update rancher version in addon config

### ISSUE THREAD

**Issue by Mithrandir2k18** [harvester/experimental-addons#24]
I've deployed rancher with v2.8.2 using the vcluster addon, but I need a rancher version that supports rke2 v1.28, so at least rancher v2.8.3.

What is the recommended way to update rancher vcluster deployments? The config field in harvester isn't editable.

**Comment by tserong** [comment]
Try changing it by editing the YAML (edit the `rancherVersion` line, and also the `image` line that specifies the k3s version if necessary)

**Comment by Mithrandir2k18** [comment]
Tried it, the harvester UI claims that it's redeploying successfully, but the rancher version still shows as 2.8.2 instead of 2.9.4 I specified. I kept the image version as rancher/k3s:v1.27.10-k3s2 as it should be compatible with rancher v2.9.4 if I read the compatibility matrix correctly.

**Comment by tserong** [comment]
I did a quick test here, just bumping the rancher version to v2.9.4, and while the Harvester UI says the deployment was successful, it's actually not. One of the pods is in CrashLoopBackOff:

```
# kubectl -n rancher-vcluster get pods | grep helm-install
helm-install-cert-manager-lfvsb-x-kube-system-x-ranc-a6c0166919   0/1     Completed          0              5d1h
helm-install-rancher-ctjsh-x-kube-system-x-rancher-vcluster       0/1     CrashLoopBackOff   14 (76s ago)   48m
helm-install-rancher-vcluster-6xpkc                               0/1     Completed          0              48m
```

...and the logs say rancher v2.9.4 isn't available:

```
# kubectl -n rancher-vcluster logs pod/helm-install-rancher-ctjsh-x-kube-system-x-rancher-vcluster|tail -n 1
Error: chart "rancher" matching v2.9.4 not found in rancher index. (try 'helm repo update'): no chart version found for rancher-v2.9.4
```

I suspect this is because rancher v2.9.4 is a Prime version release, as opposed to a Community release. The repo specified in the rancher-vcluster yaml is https://releases.rancher.com/server-charts/stable/index.yaml:

https://github.com/harvester/experimental-addons/blob/ebb49b680fa3b919c3434a3bd25ccce4e0d73f42/rancher-vcluster/rancher-vcluster.yaml#L66

The latest v2.9.x series Rancher there is v2.9.3. To get v2.9.4, you'd need to be a Prime subscriber and update the YAML to refer to the Prime repo URL, but I can't say what that is offhand.

You probably also want to update k3s to rancher/k3s:v1.27.16+k3s1 as that's the latest in the v1.27.x series, but that's unrelated -- the main issue here is whether you have access to Prime or not.

**Comment by Mithrandir2k18** [comment]
Yeah that must've been it. Now running rancher v2.9.3 on vcluster. Thank you
